src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
+type Color = 'red'|'green'|'blue';
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea: Function;
+}
+
+function roundedArea(area: number): number {
+  return Math.floor(area * 100) / 100;
+}
+
+function isTriangle(a:number, b:number, c:number):boolean {
+  const allSides = [a, b, c].sort((x:number, y:number) => x - y);
 
+  return allSides[2] < allSides[0] + allSides[1];
```

Comment:
I think that it would be better to use this function **inside the class**. 
```suggestion
  const allSides = [a, b, c].sort((x, y) => x - y);

  const isTriangle = allSides[2] < (allSides[0] + allSides[1]);
```

Reply:
it would be great to make this feature useful for every shape of the figure